draft-ietf-quic-qpack.md

```diff
@@ -1199,6 +1199,12 @@ are registered in the "HTTP/3 Error Code" registry established in {{HTTP3}}.
 
 # Static Table
 
+This table was generated by analyzing actual internet traffic in 2018 and
+manually crafting the table for optimal compression of common headers.  Due to
```

Comment:
What does "manually crafting" mean?  I think that you mean manually removing obvious redundancies and (other stuff we/Mike did).

Reply:
maybe you want to include some info from https://github.com/quicwg/base-drafts/issues/904#issuecomment-414471170

Reply:
We started from an automated query for data, then:

- Removed vendor-specific artifacts and anti-patterns as best we could
- Sorted the entries into the three tranches described in the comment; the sizes are dictated by the QPACK instruction space, but the selection of which things go in each tranche is a mix of data and judgement.